providers/desec/protocol.go

```diff
@@ -260,14 +260,6 @@ func (c *desecProvider) upsertRR(rr []resourceRecord, domain string) error {
 	return nil
 }
 
-func (c *desecProvider) deleteRR(domain, shortname, t string) error {
```

Comment:
Comment this out.  It might be needed in the future.

Reply:
Commented `deleteRR` function out